### PR TITLE
OEP submodule refactoring

### PR DESCRIPTION
**Pull request by PiotrBanuba**
Tested on macOS

### CODE REVIEW

File: cmake/copy_libs.cmake [hunk 1]
```diff
@@ -0,0 +1,73 @@
+function(copy_sdk target)
+    if (MSVC)
+        set_property(TARGET ${target} APPEND PROPERTY LINK_FLAGS /STACK:4194304)
+        set(SDK_FILE_TYPE "dll")
+
+        if(BUILD_C_API)
```
Comment (dmitrystoyanovsky) [inline cmake/copy_libs.cmake:6]: do we still need BUILD_C_API?

**Review by dmitrystoyanovsky — APPROVED** [review]

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[submodule "oep"]
+	path = oep
+	url = git@github.com:PiotrBanuba/OEP-module.git
```
Comment (nick-korotysh-bnb) [inline .gitmodules:3]: don't forget to update link after merge

File: effect_player.hpp [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        void resume() override;
+
+        void push_frame(pixel_buffer_sptr image, bnb::oep::interfaces::rotation image_orientation) override;
+
+        void draw() override;
+
+    private:
+        bnb::image_format make_bnb_image_format(pixel_buffer_sptr image, interfaces::rotation orientation);
+        bnb::yuv_format_t make_bnb_yuv_format(pixel_buffer_sptr image);
+        bnb::interfaces::pixel_format make_bnb_pixel_format(pixel_buffer_sptr image);
```
Comment (nick-korotysh-bnb) [inline effect_player.hpp:38]: these functions can be just a static non-member functions, but let's keep them as is

File: libraries/renderer/include/renderer.hpp [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 #include <bnb/types/full_image.hpp>
 
-#include "program.hpp"
```
Comment (nick-korotysh-bnb) [inline libraries/renderer/include/renderer.hpp:5]: looks like this include (and corresponding .cpp) is not required anymore (were moved somewhere), but such files still present in this example

**Review by nick-korotysh-bnb — APPROVED** [review]
approve with minor comments

File: cmake/utils.cmake [hunk 5]
```diff
@@ -4,6 +4,21 @@
 
 include(CMakeParseArguments)
```
Comment (dmitrystoyanovsky) [inline cmake/utils.cmake:5]: I don't think that file is necessary and can be removed. At least I do not see macro usage.

**Review by dmitrystoyanovsky — APPROVED** [review]
Please review the comment.

**Pull request merged**